core/trino-server-rpm/src/main/resources/dist/etc/init.d/trino

```diff
@@ -37,6 +37,9 @@ CONFIGURATION=(
     --server-log-file "${SERVER_LOG_FILE:-/var/log/trino/server.log}"
 )
 
+ulimit -Hn 131072
+ulimit -Sn 131072
```

Comment:
I expect these to fail (i.e. not take effect) unless one of `/etc/security/limits.conf` or `/etc/systemd/user.conf` or `/etc/systemd/system.conf` has a value higher than this allowed for the Trino user.

I'll try and verify in a VM.

Reply:
I think root can execute them regardless of what's in `/etc/security/limits.conf`.

Reply:
BTW I checked CentOS packages and found only one that defines limits in `/etc/security/limits.d`, which is `libvma`. But it's also the single package that has a SysV init script, not a SystemD unit file.

Reply:
This only works because CentOS (since 7) uses SystemD and systemd sets the default hard limit to a high enough value that your command doesn't fail. This is not an assumption that is safe to make.

See https://www.freedesktop.org/software/systemd/man/systemd-system.conf.html#DefaultLimitCPU=.

Reply:
Try this:

```
[root@centos hashhar]# sysctl -a | grep 'fs.nr_open'
fs.nr_open = 1048576
[root@centos hashhar]# ulimit -Hn 1048577
bash: ulimit: open files: cannot modify limit: Operation not permitted
```

Reply:
`nproc` at some point was dropped from the docs PR I linked. I'm not sure why, none of the comments there refer to that part. So we need an agreement for the right value and also put it in docs. Sounds like a separate issue.

I'm not sure what's the ask here. To set these to the greater of 131072 and current `fs.nr_open`?

Reply:
I've made changes here to only raise the limit, never lower it, and never raise it above `fs.nr_open`.